untitled5/models.py

```diff
@@ -0,0 +1,41 @@
+from django.db import models
+from djangotoolbox.fields import ListField, DictField, EmbeddedModelField
+import datetime
+
+# USER_DATA model
+class User_data(models.Model):
```

Comment:
all data fields used by backend services should be defined in one module, e.g. "user_data.py", profile_maintenance logic should be defined in another module, e.g. "profile_maintenance.py". Model.py should only have the main logic.